gst/nnstreamer/tensor_transform/tensor_transform.c

```diff
@@ -924,6 +929,26 @@ gst_tensor_transform_set_option_data (GstTensorTransform * filter)
       ret = filter->loaded = TRUE;
       break;
     }
+    case GTT_CLAMP:
+    {
+      gchar **strv = NULL;
+
+      if (!g_regex_match_simple (REGEX_CLAMP_OPTION, filter->option, 0, 0)) {
+        ml_loge
+            ("%s: clamp: \'%s\' is not valid option string: it should be in the form of [CLAMP_MIN:CLAMP_MAX]\n",
+            filter_name, filter->option);
+        break;
+      }
+
+      strv = g_strsplit (filter->option, ":", 2);
+
+      filter->data_clamp.min = g_ascii_strtod (strv[0], NULL);
+      filter->data_clamp.max = g_ascii_strtod (strv[1], NULL);
+
```

Comment:
How about adding code for checking the validity of CLAM_MIN and MAX here?

Reply:
Got it. I will add a routine to check overflow/underflow using `errno`.